gutils/unicodelibinfo.c

```diff
@@ -26,15 +26,11 @@
  * ADVISED OF THE POSSIBILITY OF SUCH DAMAGE.
  */
 
-#ifndef _GUTILS_UNICODELIBINFO_C_
-#define _GUTILS_UNICODELIBINFO_C_
```

Comment:
Any idea why this was here in the first place?

Reply:
3aaa48c1c230840045bbae158093edef53d762d3 is the context, it looks erroneously added to me.